homeassistant/helpers/template.py

```diff
@@ -2048,6 +2051,60 @@ def __bool__(self) -> bool:
         return super().__bool__()
 
 
+async def async_materialize_hass_loader(hass: HomeAssistant) -> None:
```

Comment:
The docstring says this will load custom jinja, can we make the function name more distinct?
```suggestion
async def async_load_custom_jinja(hass: HomeAssistant) -> None:
```